src/js/components/DataTable/Header.js

```diff
@@ -130,7 +130,7 @@ const Header = ({
                 pad={pad}
                 plain
                 scope="col"
-                size={size} // Initial size if specified
+                size={widths && widths[property] ? undefined : size}
```

Comment:
@ericsoderberghp I need your review here.

Reply:
looks right to me